test/SlowTests/Server/Documents/PeriodicBackup/PeriodicBackupSlowTests.cs

```diff
@@ -2103,10 +2103,10 @@ public async Task CanAbortOneTimeBackupAndRestore()
                 };
 
                 var database = await Server.ServerStore.DatabasesLandlord.TryGetOrCreateResourceStore(store.Database).ConfigureAwait(false);
+                database.PeriodicBackupRunner.ForTestingPurposesOnly().OnBackupTaskRunHoldBackupExecution = new TaskCompletionSource<object>();
```

Comment:
better to move it to a variable and complete it the end

Reply:
done